be/test/common/process_exit_test.cpp

```diff
@@ -0,0 +1,78 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this should be starrocks license hearder

Reply:
done